dolphin/alert_manager/trap_receiver.py

```diff
@@ -92,7 +146,8 @@ def _snmp_v2v3_config(self):
         }
         config.addV3User(
             self.snmp_engine, userName=constants.SNMP_USM_USER,
-            authKey=constants.SNMP_V3_AUTHKEY, privKey=constants.SNMP_V3_PRIVKEY,
+            authKey=constants.SNMP_V3_AUTHKEY,
```

Comment:
Why add snmp_v2v3_config when start trap receiver? Cause different trap source may used different security mechanism and different user.

Reply:
Yes , this mechanism will change once we support alert config handling. Anayway, created an issue to track